tc-python-tests.sh

```diff
@@ -4,22 +4,24 @@ set -xe
 
 source $(dirname "$0")/tc-tests-utils.sh
 
-pyver=$1
+pyver_full=$1
 aot_model=$2
 
-if [ -z "${pyver}" ]; then
+if [ -z "${pyver_full}" ]; then
     echo "No python version given, aborting."
     exit 1
 fi;
 
+pyver=$(echo "${pyver_full}" | cut -d':' -f1)
+
 # 2.7.x => 27
-pyver_pkg=$(echo "${pyver}" | cut -d'.' -f1,2 | tr -d '.')
+pyver_pkg=$(echo "${pyver_full}" | cut -d':' -f1 | cut -d'.' -f1,2 | tr -d '.')
```

Comment:
The first part of this, `echo "${pyver_full}" | cut -d':' -f1`, is just `$pyver`.